project/src/main/java/com/google/sps/Servlets/DeleteCommentServlet.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.*;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
```

Comment:
dont need arraylist, date, simpledateformat, multipartconfig, collectors, userservice, and userservice factory imports.